p2panda-sync-next/src/cbor.rs

```diff
@@ -0,0 +1,291 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! Utility methods to encode or decode wire protocol messages in [CBOR] format.
```

Comment:
Wondering if this is a new `p2panda-io` crate? We will need the same functionality in the discovery protocol (and also for a gossiping protocol in the future)